behave_html_pretty_formatter/html_pretty.py

```diff
@@ -221,6 +221,20 @@ def generate_feature(self, formatter):
                     with a(onclick="expander('expand_all_failed')", href="#"):
                         div("[Expand All Failed]", cls="feature-summary-row button")
 
+                if formatter.additional_info:
+                    with div(
+                        cls="feature-additional-info-container",
+                        id="additional-info",
+                        style=f"display: {summary_display}"
```

Comment:
This line is wrong. It makes additional info never show, if there is "show_summary = false" in config. Please drop this line.

Reply:
And also short mention in `CHANGES` what is new in 1.3. Thanks a lot.